server/planning/events/events.py

```diff
@@ -192,18 +192,21 @@ def on_create(self, docs):
             if event["dates"].get("recurring_rule", None):
                 event["dates"]["start"] = get_date(event["dates"]["start"])
                 event["dates"]["end"] = get_date(event["dates"]["end"])
-                recurring_events = generate_recurring_events(event)
-                generated_events.extend(recurring_events)
-                # remove the event that contains the recurring rule. We don't need it anymore
-                docs.remove(event)
-
-                # Set the current Event to the first Event in the new series
-                # This will make sure the ID of the Event can be used when
-                # using 'event' from here on, such as when linking to a Planning item
-                event = recurring_events[0]
-                # And set the Planning Item from the original
-                # (generate_recurring_events removes this field)
-                event["_planning_item"] = planning_item
+
+                # If _created_externally is true, generate_recurring_events is restricted.
+                if event["dates"]["recurring_rule"].get("_created_externally", None) and not event["dates"]["recurring_rule"]["_created_externally"]:
```

Comment:
This would need to go further up, line 192:
```python
if event["dates"].get("recurring_rule", None) and not event["dates"]["recurring_rule"].get("_created_externally", False):
```

There is also no need to double check `"_created_externally"`, the following will suffice:
```python
not event["dates"]["recurring_rule"].get("_created_externally", False)
```